{{cookiecutter.project_slug}}/src/{{cookiecutter.project_slug}}/data_management/task_clean_data.R

```diff
@@ -0,0 +1,22 @@
+# ======================================================================================
```

Comment:
What is relation between task_clean_data.R and clean_data.r?

Reply:
My idea was to have a similar structure across languages. In the Python example, you will find a `clean_data.py`, containing functions relevant for the data cleaning, and a `task_data_management.py`, containing a `clean_data` task that only does IO-related things and calling functions from `clean_data.py`. I tried to achieve a similar relation for `task_clean_data.r` and `clean_data.r`. In particular, I wanted `clean_data.r` to work as a module that exports functions and not as a script that executes commands. One could move the function definitions to the beginning of `task_clean_data.r`; however, if one has many/long functions then `task_clean_data.r` would become a bit cluttered.

Reply:
My thinking would be a bit different (more from a research project perspective, not so much about having a nice package). Everything that is a "task" should be handled by pytask. No need for this thing in R, I'd think that such stuff will mostly be confusing people.

That is, I'd think about pytask calling an R script just like it would call a (Python) function.

> One could move the function definitions to the beginning of task_clean_data.r; however, if one has many/long functions then task_clean_data.r would become a bit cluttered.

I believe the question of how to best divide code up across modules is independent of pytask. In any case, not a problem in this example.

So, could you implement what you suggest, but in `clean_data.r`, deleting `task_clean_data.r`? And similarly for the other directory. 

That'll be it for good, then :wink:

Reply:
Just to be clear, that means I simply append the contents of `task_clean_data.r` to `clean_data.r`?

Reply:
Precisely. Thanks!

Reply:
Excellent, thank you!